mlflow/utils/_capture_modules.py

```diff
@@ -112,6 +112,7 @@ def store_imported_modules(cap_cm, model_path, flavor, output_file):
         mlflow_model = Model.load(model_path)
         pyfunc_conf = mlflow_model.flavors.get(mlflow.pyfunc.FLAVOR_NAME)
         input_example = mlflow_model.load_input_example(model_path)
+        params = mlflow_model.load_input_example_params(model_path)
```

Comment:
Can we only read file one pass and get both `input_example` and `params` ?

Reply:
That was the weird behavior I asked in last standup. In that case, to make the function backward compatible, we need to return `single input_example` when there're no params, or `a tuple of input_example and params` when there're params... That's actually even worse than the current one 😓

Reply:
That was the weird behavior I asked in last standup. In that case, to make the function backward compatible, we need to return `single input_example` when there're no params, or `a tuple of input_example and params` when there're params... That's actually even worse than the current one 😓

Reply:
ok. we can keep current code, not a big deal.